src/server/service/kv.rs

```diff
@@ -133,8 +130,8 @@ impl<T: RaftStoreRouter + 'static> Service<T> {
             snap_scheduler: snap_scheduler,
             token: Arc::new(AtomicUsize::new(1)),
             recursion_limit: recursion_limit,
-            stream_channel_size: stream_channel_size,
             metrics: Metrics::new(),
+            stream_channel_size,
```

Comment:
Forget to init?

Reply:
This is a syntax sugar: SomeStruct { foo, bar } is equally SomeStruct { foo: foo, bar: bar }

Reply:
Please keep style consistency with other fields.